domain/build.gradle

```diff
@@ -29,4 +29,7 @@ repositories {
 
 dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
+
```

Comment:
Please move this before test dependencies to be consistent with all other gradle files